homework/index.js

```diff
@@ -30,13 +30,75 @@
     return elem;
   }
 
+  function pageConstructor(data) {
```

Comment:
`data` or `dt` is a name for a general use, for me if you will make me a choice I will use `data` over `dt`, because it refers to an unknown data somehow.

for a general naming what I think about a name usually is what kind of information it contains.
in this case it's an information about repositories.